CondTools/RunInfo/src/LHCInfoPerLSPopConSourceHandler.cc

```diff
@@ -469,8 +504,7 @@ void LHCInfoPerLSPopConSourceHandler::getNewObjects() {
     }
 
 
-    size_t nlumi = getLumiData(oms, lhcFill, startSampleTime, endSampleTime);
-    edm::LogInfo(m_name) << "Found " << nlumi << " lumisections during the fill " << lhcFill;
+    getLumiData(oms, lhcFill, startSampleTime, endSampleTime);
```

Comment:
Can you explain the reason why the printout is no longer executed ?

Reply:
It's been moved to line 142, inside getLumiData https://github.com/CTPPS/cmssw/pull/62/files#diff-f26dd73d4300142b911cc877df1bcf2313392b1a0835c7a004d615fdc3f887b8R142
I've decided that getLumiData shouldn't return the number of lumisections it got from OMS but the number of buffered lumisections and the debug info about how many lumisections it has got from OMS should be inside the method.